patches/chromium/make_keychainpassword_service_name_and_service_name_mutable_strings.patch

```diff
@@ -0,0 +1,193 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Wed, 27 Jan 2021 15:43:35 -0800
+Subject: make KeychainPassword::service_name, kApplicationName and
+ service_name mutable strings
+
+This should be upstreamable, it's just a type change that makes it
```

Comment:
Can you try upstreaming this before we land please? :)

Reply:
This patch comment isn't completely accurate since I added the patches required for linux.  The macOS bit in isolation I can for sure try to upstream, the linux bits probably need a bit of thinking through before trying to upstream 🤔

Reply:
Both patches have been completely upstreamed and accepted